### ISSUE TITLE
User-merge command

### ISSUE THREAD

**Issue by OndraM** [lmc-eu/matej-client-php#23]
- `Lmc\Matej\Model\Command\UserMerge`
- Static named constructor with helpful name (so you are sure which user is source and which target). Something like `UserMerge::toTargetUserMergeSourceUser($targetUser, $sourceUser)` :-).

Once the command is done, in separate commit (or in separate pull request) the EventsRequestBuilder should be extended by `addUserMerge` and `addUserMerges` methods.
